### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by eadle** [siphontv/p2ptv#3]
- verify memory leak fixes #7
- fix sequential streams
- basic p2p network structure
- p2p statistics
- add media player example to project
- add support for big endian clients

_Bugs:_
- server doesn't handle end of stream properly (occasionally emits malformed stream)
- server crashes when media segment has more than 256 chunks

**Comment by OliCat** [comment]
Ok !!!
I were work on other projects the last two week. Tests and feedbacks this week-end :-)
Very big thanks for your work !

OliCat

**Comment by eadle** [comment]
Haha, sounds good.

Probably won't start working on P2P delivery for another month. I'm currently working on the media player API ([demo](http://siphon.tv/embed)). Would this be something you'd expect to be included in the project? Like an embed.html or something?

**Comment by OliCat** [comment]
Woooo ! Pretty cool this embeded player ! This project is very exiting ;-)

**Comment by eadle** [comment]
Thanks. Still needs a lot of work though.

**Comment by OliCat** [comment]
Yes, i know. Nobody have contacted you to work with you on distributed network ?
Concerning my test, i have to install via npm or i download the source here ?

**Comment by eadle** [comment]
You can test with current version on npm. I haven't decided on a policy for what I commit to github, so for now just assume that you should use the latest version on npm.

**Comment by OliCat** [comment]
I confirm :-) It's now ok for me with "test.mp4" media file ! http://stream-lat.org:8181

**Comment by eadle** [comment]
lol. Works for me. Nicely done.

**Comment by OliCat** [comment]
You're song choice is great ;-)

**Comment by eadle** [comment]
I laughed. Feel free to post any bugs you encounter. It's 5AM here. Have fun mang.

**Comment by OliCat** [comment]
Thanks. Bye

**Comment by OliCat** [comment]
Hello,

about distributed architecture, a friend told me that sails.js (http://sailsjs.org/features) could be an interesting option.

Have a good day.

**Comment by eadle** [comment]
Sorry man, probably won't be working on this for a while.
